eng/performance/benchmark_jobs.yml

```diff
@@ -123,11 +123,20 @@ jobs:
               ${{ channel }}:
                 _Channel: ${{ channel }}
                 _Configs: CompilationMode=Tiered RunKind="${{ parameters.kind }}"
-                _PerfLabArguments: $(PerfLabArguments)
+                _PerfLabArguments: $(PerfLabArguments) # not necessary?
                 _BuildConfig: ${{ parameters.architecture }}_$(_Channel)_${{ parameters.kind }} # needs to be unique to avoid logs overwriting in mc.dot.net
         steps:
         - checkout: self
           clean: true
+        - task: UsePythonVersion@0
+          inputs:
+            versionSpec: '3.10'
```

Comment:
Since we install python in at least 2 places (here and scnearios.yml), it may be beneficial to see if we can pull the version into a single variable or version in a file so we only have to update it in one place.

Reply:
I've changed it now so that we just use python 3.6 actually, but dataclasses are supported by installing the dataclasses pip package

Reply:
Is dataclasses something we should add to the requirements.txt file?